packages/db-postgres/src/findVersions.ts

```diff
@@ -23,7 +23,7 @@ export const findVersions: FindVersions = async function findVersions(
   },
 ) {
   const collectionConfig: SanitizedCollectionConfig = this.payload.collections[collection].config
-  const sort = typeof sortArg === 'string' ? sortArg : collectionConfig.defaultSort
+  const sort = sortArg || '-updatedAt'
```

Comment:
This should have `collectionConfig.defaultSort` as a fallback instead of `-updatedAt` still shouldn't it?